apps/mochi-web/components/settings/notification/NotificationPage.tsx

```diff
@@ -153,9 +160,9 @@ export function NotificationPage() {
       website: platforms?.includes('website'),
       ...flags,
     })
-  }, [isLoading, reset, settings])
+  }, [isFirstLoading, reset, settings])
 
-  if (isLoading) return <Skeleton />
+  if (isFirstLoading) return <Skeleton />
```

Comment:
I’m not sure why the loading UI is not visible in the preview. Is there a possibility that it never appears?